Pilot1/P1B2/p1b2_default_model_pytorch.txt

```diff
@@ -0,0 +1,28 @@
+[Global_Params]
```

Comment:
Do we need this separately for the pytorch model? Can we reuse p1b2_default_model.txt?

Reply:
The original point of the default model files was that a correctly written benchmark (in any language) would read the same default_model file and instantiate the same model, so it should indeed be reading p1b2_default_model.txt.

Reply:
I have updated with using the p1b2_default_model.txt file and deleted the pytorch model file.